workspace.code-workspace

```diff
@@ -91,13 +91,14 @@
       "source.fixAll.eslint": true
     },
     "cSpell.words": [
```

Comment:
are you even using cSpell.words? 🤔

Reply:
I'm using it. But indeed, we should add it to our recommended plugins. It is awesome! Been adding words in our config (using `ctrl+.` and choosing "add this word to workspace"). 

![image](https://user-images.githubusercontent.com/1828233/117408263-f681bd00-af0f-11eb-8bd6-e4bdec365d6b.png)